integ-test/src/test/java/org/opensearch/sql/sql/DateTimeFunctionIT.java

```diff
@@ -817,6 +817,18 @@ public void testSecondFunctionAliasesReturnTheSameResults() throws IOException {
     result1.getJSONArray("datarows").similar(result2.getJSONArray("datarows"));
   }
 
+  @Test
+  public void testStrToDate() throws IOException {
+    JSONObject result = executeQuery(
+        String.format("SELECT str_to_date(CAST(birthdate AS STRING),"
+                + " '%%Y-%%m-%%d %%h:%%i:%%s') FROM %s LIMIT 2",
+            TEST_INDEX_BANK));
+    verifyDataRows(result,
+        rows("2017-10-23 00:00:00"),
+        rows("2017-11-20 00:00:00")
+    );
+  }
+
```

Comment:
Might be nice to add IT where the string parameter is not an acceptable format

Reply:
Added in https://github.com/Bit-Quill/opensearch-project-sql/pull/231/commits/026c830f001a3b8a463cbb1cf8dab7bd6689d054